CMakeLists.txt

```diff
@@ -84,7 +84,8 @@ if (ISLE_UBSAN)
   add_link_options(-fsanitize=undefined)
 endif()
 
-#if (NOT WIN32)
+# Force reported render mods from MiniWin
+add_compile_definitions(MINIWIN_PIXELFORMAT=SDL_PIXELFORMAT_INDEX8)
```

Comment:
Let's move this below `add_library(miniwin)` and use `target_compile_definitions`.
Or move this to `miniwin_p.h`.